src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+export const CommentInfo = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {name}
+        </strong>
+
+        {' by '}
```

Comment:
- it's a bad practice to add the spaces this way
- you can put this text to tag and set margins and padding 
please check other places